scripts/GitHubMergeBranches.ps1

```diff
@@ -272,8 +268,29 @@ try {
         | select -First 1
 
     if ($matchingPr) {
+        $prUpdatedSuccess = $false
+
+        try {
+            if ($PSCmdlet.ShouldProcess("Update remote branch $mergeBranchName on $remoteName")) {
+                Invoke-Block { & git push $remoteName "${mergeBranchName}:${mergeBranchName}" }
+            }
+            $prUpdatedSuccess = $true
+        }
+        catch {
+            Write-Warning "Failed to update existing PR"
+        }
+
+        $prMessage = if ($prUpdatedSuccess) {
+            "This pull request has been updated.`n`n$committersList"
+        } else {
+            @"
+:x: Uh oh, this pull request could not be updated automatically. New commits were pushed to $HeadBranch, but I could not automatically push those to $mergeBranchName to update this PR.
+You may need to fix this problem by merging branches with this PR. Contact the ASP.NET Core Engineering if you are not sure what to do about this.
```

Comment:
nit:

Contact the ASP.NET Core Engineering **Team**...

or

Contact ~~the~~ ASP.NET Core Engineering ...

Reply:
good catch. updated.